bwrap.xml

```diff
@@ -336,6 +338,9 @@
       <listitem>
         <para>Copy from the file descriptor <arg choice="plain">FD</arg> to
           <arg choice="plain">DEST</arg>.</para>
+        <para>Note that, unlike <option>--bind-data</option>, if
+          <arg choice="plain">DEST</arg> is visible outside of the created mount
+          namespace, the effect is permanent.</para>
```

Comment:
I think it's worth calling out `--file` specifically, because it's the only one that will silently overwrite an existing file, but I'd prefer it more like perhaps this:

```
        <para>If <arg choice="plain">DEST</arg> already exists, it will be overwritten.
          Unlike <option>--bind-data</option>, if <arg choice="plain">DEST</arg> or
          a parent directory is a bind-mount from outside the sandbox,
          this may overwrite an existing file outside the sandbox.</para>
```

Would that have given you the right expectations?

Reply:
Yes, that sounds great. Shall I apply it to this PR?

Reply:
Yes please!

Reply:
Done!